python/aitemplate/testing/detect_target.py

```diff
@@ -38,6 +38,8 @@ def _detect_cuda():
         stdout = stdout.decode("utf-8")
         if "A100" in stdout or "RTX 30" in stdout or "A30" in stdout or "A10" in stdout:
             return "80"
+        if "V100" in stdout:
+            return "70"            
```

Comment:
Would you be able to disable the V100? since other ops may not work

Reply:
Unfortunately, I do not have access to A100.  Will find one to test. But all of our prod is based on V100 so we have to support V100 and limit ourselves to the set of kernels we can run there, for now.

Reply:
Thankfully the gating is removed in the main branch now!